docker-compose.yml

```diff
@@ -30,6 +30,7 @@ services:
       - "polis-net"
     ports:
       - "5000:5000"
+      - "9229:9229"
```

Comment:
@colinmegill @patcon @micahstubbs As mentioned [here](https://github.com/compdemocracy/polis/pull/961#discussion_r768206590), this _should_ be unnecessary given the `docker-compose.dev.yml`, and needs to be removed as part of the process of moving all of the development related `docker-compose.yml` config over to `*.dev.yml` (in order to get our docker compose setup production ready).